src/Benchmarks/Benchmarks.csproj

```diff
@@ -19,6 +19,7 @@
     <!-- Prevent the SDK from validating the supported TFM. -->
     <NETCoreAppMaximumVersion>99.9</NETCoreAppMaximumVersion>
     <NoWarn>0618;NU1605</NoWarn>
+    <AssemblyOriginatorKeyFile>AspNetCore.snk</AssemblyOriginatorKeyFile>
```

Comment:
REVIEW: This change doesn't seem to cause `dotnet build` to strong name the assembly, but I created the PR anyway to encourage someone to show me the correct way to do this without porting this project to arcade.